scheduler/scheduler-server/src/test/java/functionaltests/rm/TestRMReconnectioWhileRunning.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package functionaltests.rm;
+
+import functionaltests.utils.*;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.objectweb.proactive.core.config.ProActiveConfiguration;
+import org.ow2.proactive.resourcemanager.RMFactory;
+import org.ow2.proactive.scheduler.common.job.JobId;
+import org.ow2.proactive.scheduler.common.job.JobInfo;
+import org.ow2.proactive.scheduler.common.job.JobResult;
+import org.ow2.proactive.scheduler.common.job.JobStatus;
+
+import java.io.File;
+import java.net.URL;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+
+
+public class TestRMReconnectioWhileRunning extends MultipleRMTBase {
+
+    static final Logger logger = Logger.getLogger(TestRMReconnectioWhileRunning.class);
+
+    private SchedulerTHelper schedulerHelper;
+
+    private static URL runningJob = TestRMReconnectioWhileRunning.class.getResource("/functionaltests/descriptors/Job_20s.xml");
+    private static URL runningJob1 = TestRMReconnectioWhileRunning.class.getResource("/functionaltests/descriptors/Job_20s-1.xml");
+    private static URL runningJob2 = TestRMReconnectioWhileRunning.class.getResource("/functionaltests/descriptors/Job_20s-2.xml");
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        if (TestScheduler.isStarted()) {
+            SchedulerTHelper.log("Killing previous scheduler.");
+            TestScheduler.kill();
+        }
+        initConfigs();
+    }
+
+
+    @Ignore
+    @Test
+    public void test() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1607)

Reply:
As sonar said, fix or remove the @ignore test

Reply:
If I remove this test will fail constantly, do we want it? I mean I dont have idea how to fix it yet